src/constants.rs

```diff
@@ -7,4 +7,11 @@ pub mod ethereum {
 
     /// zkSync smart contract address.
     pub const ZK_SYNC_ADDR: &str = "0x32400084C286CF3E17e7B677ea9583e60a000324";
+
+    /// The path to the initial state file.
+    pub const INITAL_STATE_PATH: &str = "InitialState.csv";
+
+    // TODO: Configure via env / CLI.
+    /// The path of the state file.
+    pub const STATE_FILE_PATH: &str = "StateSnapshot.json";
```

Comment:
```suggestion
}

pub mod storage {
    /// The path to the initial state file.
    pub const INITAL_STATE_PATH: &str = "InitialState.csv";

    // TODO: Configure via env / CLI.
    /// The path of the state file.
    pub const STATE_FILE_PATH: &str = "StateSnapshot.json";
```

My thinking with the `ethereum` submodule was to namespace constants appropriately. These filenames are related to storage so maybe create `storage` submodule similarly?